skrypty/team/core.lua

```diff
@@ -343,8 +343,18 @@ function ateam:print_obj_team(id, obj)
             cecho(scripts.ui.states_window_name, "<white:team_console_bg>[  ]")
         end
 
+        -- team lead
+        local str_lead = " "
+        if obj["team_leader"] and ateam.options.leader_indicator_symbol ~= " " then
+            if string.len(ateam.options.leader_indicator_symbol) > 1 then
```

Comment:
If you use `""` this check will become redundant.

Reply:
ale wtedy musze dac
350: `elseif string.len(ateam.options.leader_indicator_symbol) > 0`

zdecyduj jak chcesz :)

Reply:
te dwa ify sa niepotrzebne.
Zanicjuj tą zmienną z wartością `""` i nie sprawdzaj dlugosci tylko używaj zawsze.

Reply:
Dodalem to obcinanie dlugosci, bo nie wiedzialem jak zrobic character type. W sumie jest zbedne, prawda.

Jesli nie bede sprawdzal wogole, to dla prowadzacego druzyne, nie bedzie spacji miedzy paskiem HP a obj["desc"], przez to:
```     cecho(scripts.ui.states_window_name, "<"..ateam.options.bracket_hp_color..":team_console_bg>"..ateam.options.bracket_symbol_left.."<reset>" .. states[obj["hp"]] .. "<"..ateam.options.bracket_hp_color..":team_console_bg>"..ateam.options.bracket_symbol_right..str_lead)```

Reply:
Ale nie jest to potrzebne, jak ktoś będzie miał fantazję to subie ustawi `General `, a ktoś będzie chciał dać `+` to da.
Więc niech zawsze będzie spacja po nawiasie, tak jak było. A tutaj nie rób żadnego cięcia.

Reply:
czolem,
taki jest expected behavior:
![Screenshot from 2022-01-28 16-32-01](https://user-images.githubusercontent.com/11772152/151585221-76e96f07-c38c-4701-9a23-13ef6f10cc16.png)

jesli zostawie spacje, tak jak proponujesz, to nie bede mial imion aligned w kolumnie

Reply:
alternatywnie moge appendowac ten znaczek na koncu stringa, przed lista "<-" atakujacych

Reply:
poprawilem wg rekomendacji. usunalem sprawdzanie dlugosci. tekst jest z lewej i nie ma juz alignmentu imion.